src/utils/message_builder.rs

```diff
@@ -723,6 +727,190 @@ impl fmt::Display for MessageBuilder {
     }
 }
 
+
+/// Content modifier enum
```

Comment:
This should be a bit more descriptive; maybe "Modifiers for message builder content pushes."?